packages/pn-personagiuridica-webapp/src/models/ApiKeys.ts

```diff
@@ -0,0 +1,106 @@
+export enum ApiKeyStatus {
```

Comment:
We have to do a check with Carlotta, but I think that if the models are the same of generated ones, this file can be removed